<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialogBox</class>
 <widget class="QDialog" name="PreferencesDialogBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Application Paths</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dice System</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diagnotic</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="General">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Color</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Fog Of War:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="ColorButton" name="m_fogColor">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Opacity Value:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="m_opacitySlider">
                <property name="minimum">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <number>250</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="m_opacitySpin">
                <property name="minimum">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <number>250</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Only apply on future maps.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Background</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Color:</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="ColorButton" name="m_bgColorPush">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Image:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="FileDirChooser" name="m_backgroundImage" native="true"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Behaviour</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="m_checkUpdate">
              <property name="text">
               <string>Check for Update</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_pictureAdjust">
              <property name="text">
               <string>Pictures fit their Windows (at the opening)</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Default Permission:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="m_defaultMapModeCombo"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="paths">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Directories</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Music (GM) :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Music (Player) :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Pictures :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Map :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Scenarios :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Minutes :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Chat :</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="FileDirChooser" name="m_musicDirGM" native="true"/>
              </item>
              <item>
               <widget class="FileDirChooser" name="m_musicDirPlayer" native="true"/>
              </item>
              <item>
               <widget class="FileDirChooser" name="m_pictureDir" native="true"/>
              </item>
              <item>
               <widget class="FileDirChooser" name="m_mapDir" native="true"/>
              </item>
              <item>
               <widget class="FileDirChooser" name="m_scenarioDir" native="true"/>
              </item>
              <item>
               <widget class="FileDirChooser" name="m_minuteDir" native="true"/>
              </item>
              <item>
               <widget class="FileDirChooser" name="m_chatDir" native="true"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="DiceSystem">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QTableView" name="m_tableViewAlias">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QPushButton" name="m_addDiceAliasAct">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_delDiceAliasAct">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_upDiceAliasAct">
             <property name="text">
              <string>Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_downDiceAliasAct">
             <property name="text">
              <string>Down</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_topDiceAliasAct">
             <property name="text">
              <string>Top</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_bottomDiceAliasAct">
             <property name="text">
              <string>Bottom</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Diagnotic">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QTextEdit" name="m_diagDisplay">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>278</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="m_startDiag">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileDirChooser</class>
   <extends>QWidget</extends>
   <header location="global">widgets/filedirchooser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>widgets/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialogBox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialogBox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_opacitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_opacitySpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_opacitySpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_opacitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
